src/main/java/net/mcreator/ui/dialogs/workspace/NeoForgeWorkspacePanel.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2020 Pylo and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.dialogs.workspace;
+
+import net.mcreator.generator.Generator;
+import net.mcreator.generator.GeneratorConfiguration;
+import net.mcreator.generator.GeneratorFlavor;
+import net.mcreator.ui.component.JEmptyBox;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class NeoForgeWorkspacePanel extends AbstractWorkspacePanel {

```

Comment:
I thought of something. As we're having multiple modding APIs for the Java edition that can be switched from one to another, wouldn't make sense to make a `JavaAPIWorkspacePanel` class and create an instance for each Java modding API and pass the `GeneratorFlavor` through an argument for the few texts that would need to reference the API's name (similarly to blocks and items in Minecraft) 🤔

Reply:
I didn't mean to have a single button. What I meant was instead of having multiple classes doing the same thing, making one general class that co taons all the code, but for each APU we want/need, we just create a new instance (like currently, but inztrad of the specific current class, it's the new general class) and passing the GeneratorFlavor into the arguments (and we use it for string texts).